README.md

```diff
@@ -135,6 +135,7 @@ This repository only contains the server implementation. Here are some known cli
 * [Oni](https://github.com/onivim/oni/wiki/Language-Support#java) : modern modal editing - powered by Neovim.
 * [LSP Java](https://github.com/emacs-lsp/lsp-java) : a Java LSP client for Emacs
 * [Eclipse Theia](https://github.com/theia-ide/theia) : Theia is a cloud & desktop IDE framework implemented in TypeScript
+* [EclipseIDE JDT.ls](https://github.com/redhat-developer/eclipseide-jdtls/) : an extenstion for Eclipse IDE
```

Comment:
Should it be "EclipseIDE" or "Eclipse IDE" ?  I would make JDT.ls -> JDT.LS . Otherwise this looks fine to me.

Reply:
Although I'm open to "JDT.ls" if this is your preference.